jenkins-jobs/scripts/check-contributors.sh

```diff
@@ -1,30 +1,48 @@
 #!/bin/bash
 
+DIR="$HOME/debezium-copyright-list"
 COPYRIGHT="COPYRIGHT.txt"
-CONTRIBUTOR_NAMES="/tmp/repository-CONTRIBUTOR-NAMES.txt"
-CONTRIBUTORS="/tmp/repository-CONTRIBUTORS.txt"
-
 FILTERS="jenkins-jobs/scripts/config/FilteredNames.txt"
 ALIASES="jenkins-jobs/scripts/config/Aliases.txt"
 
-git log --pretty=format:"%an" . | sort | uniq > $CONTRIBUTOR_NAMES
-git log --pretty=format:"%an|%ae" . | sort | uniq > $CONTRIBUTORS
+mkdir -p $DIR
+
+CONTRIBUTOR_NAMES="$DIR/repository-CONTRIBUTOR-NAMES.txt"
+CONTRIBUTORS="$DIR/repository-CONTRIBUTORS.txt"
+COPYRIGHT_ALIASES="$DIR/Aliases.txt"
+COPYRIGHT_FILTERS="$DIR/FilteredNames.txt"
+COPYRIGHT_DEBEZIUM_REPOS="$DIR/COPYRIGHT.txt"
+
+cp $ALIASES $FILTERS $COPYRIGHT $DIR && cd $DIR
+
+declare -a DEBEZIUM_REPOS=("debezium" "debezium-connector-db2" "debezium-connector-cassandra" "debezium-connector-vitess" "docker-images")
 
-rc=0
-while read LINE
+for REPO in "${DEBEZIUM_REPOS[@]}";
 do
+  echo "========================================================================"
+  echo "Running check for $REPO repository"
+  echo "========================================================================"
+
+  git clone --quiet https://github.com/debezium/$REPO.git
```

Comment:
Please expect that the repos are already checked-out as this is repsonsibility of release job

Reply:
ok, let me explain. If you've noticed the script did not do any repo work as it was expected the repo is already available.

Please expect this environment. There is a common job directory, this job directory contain directories with main debezium repo and repos with other connectors. This is someone else repsonsibility.

You should just work with these directories. The list of dirs could but does not need to be passed as the script params.

So in general there will be no git working stuff as it was done by Jenkins or developer and you'll just the dirs already prepared

Reply:
> If you've noticed the script did not do any repo work as it was expected the repo is already available.

@jpechane  Do you mean the `git clone`? It actually does the git clone for each repo, I have just added `--quiet` tag. 

If not any git working stuff, what needs to be actually done for this issue?

Reply:
No, the current script does not do any `git` operations but `git log`. And it should be done that way. You should just expect the repos were already checked out.

Reply:
I assume the current script only checks copyright for the `debezium` repo, right? 
If the repos are already checkout then there is nothing else to do, cause the script would be running for all of repos isn;t?